mecra.csl

```diff
@@ -0,0 +1,1467 @@
+                <label form="verb-short" text-case="lowercase" suffix=" "/>
+                <name and="symbol"/>
+              </names>
+            </group>
+          </if>
+        </choose>
+      </group>
+      <names variable="editor collection-editor editorial-director director" delimiter=", ">
+        <label form="verb-short" text-case="lowercase" suffix=" "/>
+        <name and="symbol"/>
+      </names>
+      <choose>
+        <if type="chapter" match="none">
+          <names variable="translator">
+            <label form="verb-short" suffix=" "/>
+            <name/>
+          </names>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="secondary-contributors-note">
+    <choose>
+      <if type="paper-conference entry-dictionary entry-encyclopedia chapter" match="none">
+        <text macro="editor-translator"/>
+      </if>
+      <else-if type="chapter" match="any">
+        <text term="translator" form="verb-short" suffix=" "/>
+        <names variable="translator"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="container-contributors-note">
+    <choose>
+      <if type="paper-conference entry-dictionary entry-encyclopedia" match="any">
+        <text macro="editor-translator"/>
+      </if>
+      <else-if type="chapter" match="any">
+        <text macro="editor-translator"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter paper-conference entry-encyclopedia entry-dictionary" match="none">
+        <group suffix=",">
+          <names variable="collection-editor editorial-director director">
+            <label form="verb-short" text-case="capitalize-first" suffix=" "/>
+            <name and="symbol"/>
+          </names>
+        </group>
+        <group delimiter="; " suffix=",">
+          <choose>
+            <if match="all" variable="editor translator">
+              <names variable="editor" delimiter=". ">
+                <label form="short" plural="never" text-case="capitalize-first" suffix=" "/>
+                <name and="symbol"/>
+              </names>
+              <names variable="translator">
+                <label form="short" plural="never" suffix=" "/>
+                <name and="symbol"/>
+              </names>
+            </if>
+            <else-if match="none" variable="editor">
+              <choose>
+                <if match="any" variable="translator">
+                  <names variable="translator">
+                    <label form="short" text-case="title" suffix=" "/>
+                    <name and="symbol"/>
+                  </names>
+                </if>
+              </choose>
+            </else-if>
+          </choose>
+        </group>
+      </if>
+      <else-if type="chapter" match="any">
+        <group delimiter=" " suffix=", ">
+          <text term="translator" form="verb-short" text-case="capitalize-first"/>
```

Comment:
ditto to the above -- this should be done with label within `names`, not with a term.